src/patternfly/components/Slider/examples/Slider.md

```diff
@@ -0,0 +1,267 @@
+---
+id: 'Slider'
+beta: true
+section: components
+cssPrefix: pf-c-slider
+---
+
+## Examples
+### Discrete
+```hbs
+{{#> slider
+  slider--value-min="0"
+  slider--value-max="8"
+  slider--value-now="5"
+  slider--attribute='style="--pf-c-slider--value: 62.5%;"'
+  }}
+  {{#> slider-main}}
+    {{> slider-rail}}
+    {{#> slider-steps}}
+      {{> slider-step slider-step--Left="0%" slider-step--label="0" slider-step--IsActive="true"}}
```

Comment:
does the left value always have to be in % or could it be numerical

Reply:
Could the left value and the label equal the same here?

Reply:
The left position will always have to be a percentage even if the value of the step/tick/label is a different number, because it's used to declare the `left` property in the layout, so it will need to be a percentage.